src/test/java/harness/utils/CleanupTestUserWorkspaces.java

```diff
@@ -61,7 +61,8 @@ private static void deleteWorkspaces(TestUsers testUser, boolean isDryRun) throw
                 .filter(user -> user.email.equalsIgnoreCase(testUser.email))
                 .findAny();
         // skip deleting if the test user is not an owner
-        if (workspaceUser.isEmpty() || !workspaceUser.get().roles.contains(IamRole.OWNER)) {
+        if (workspaceUser.isEmpty()
+            || !workspaceUser.get().roles.contains(WorkspaceUser.Role.OWNER)) {
```

Comment:
nit. Instead of checking empty first you can do map/contains/orElse(false).

Reply:
I find having it written out slightly easier to follow. just personal preference.